src/main/java/com/intuit/graphql/orchestrator/schema/transform/FederationTransformerPostMerge.java

```diff
@@ -68,6 +68,8 @@ public UnifiedXtextGraph transform(UnifiedXtextGraph unifiedXtextGraph) {
                       .dataFetcherType(DataFetcherContext.DataFetcherType.ENTITY_FETCHER)
                       .entityExtensionMetadata(entityExtensionMetadata)
                       .build();
+
+              unifiedXtextGraph.getEntityFetcherContexts().add(fieldContext);
```

Comment:
I might have missed it, where do we used `fieldContext` after adding?

Reply:
Oops; that is some old code that needs to be removed